resources/views/checkout/partials/sections/login.blade.php

```diff
@@ -1,34 +1,46 @@
+            <template v-if="!loggedIn">
+                <x-rapidez-ct::input
+                    label="E-mail address"
+                    name="email"
+                    type="email"
+                    v-bind:value="email"
+                    v-on:input="loginInputChange"
+                    v-on:blur="$root.guestEmail = email; if(!password) { go() }"
+                    required
+                    placeholder="Enter your e-mail address"
+                />
 
-            <x-rapidez-ct::input
-                label="E-mailaddress"
-                name="email"
-                type="email"
-                v-bind:value="email"
-                v-on:input="loginInputChange"
-                required
-                placeholder="Enter your e-mailaddres"
-            />
+                <x-rapidez-ct::input
+                    v-if="!emailAvailable"
+                    label="Password"
+                    name="password"
+                    type="password"
+                    ref="password"
+                    v-on:input="loginInputChange"
+                    required
+                />
 
-            <x-rapidez-ct::input
-                v-if="!emailAvailable"
-                label="Password"
-                name="password"
-                type="password"
-                ref="password"
-                v-on:input="loginInputChange"
-                required
-            />
+                <p v-if="!emailAvailable" class="self-end">Je hebt al een account met dit e-mailadres. Login om verder te gaan</p>
+                <p v-else class="self-end">Naar dit e-mailadres sturen wij uw bestelling bevestiging. Ook controleren we of u al een account hebt zodat u sneller kunt afrekenen.</p>
 
-            <p v-if="!emailAvailable" class="self-end">Je hebt al een account met dit e-mailadres. Login om verder te gaan</p>
-            <p else class="self-end">Naar dit e-mailadres sturen wij uw bestelling bevestiging. Ook controleren we of u al een account hebt zodat u sneller kunt afrekenen.</p>
-
-            <x-rapidez-ct::button.accent v-if="!emailAvailable" type="submit" dusk="continue">
-                @lang('Login')
-            </x-rapidez-ct::button.accent>
-        </form>
+                <x-rapidez-ct::button.accent v-if="!emailAvailable" dusk="continue" v-on:click.prevent="go">
+                    @lang('Login')
+                </x-rapidez-ct::button.accent>
+            </template>
+            <template v-else>
+                <div class="h-[52px] px-4 flex items-center rounded border border-border bg-inactive-200">
+                    <x-heroicon-o-user-circle class="h-[24px] mr-[10px]"/>
+                    <span v-text="$root.user?.email"></span>
+                </div>
+                <div>
+                    <x-rapidez-ct::title.sm>@lang('Welcome back') @{{ $root.user?.firstname }}!</x-rapidez-ct::title.sm>
+                    @lang('Blah blah blah something something')
```

Comment:
@jordythevulder Because we're (almost certainly) not putting a log-out button in the checkout as that makes you lose your cart, what text should be here?

Reply:
Uhm, so what happends if you trying to buy stuff but see that you are logged in with a different account, and can't logout anywhere. Also if i'm already logged in on an account which is not mine, that would also mean that i won't get a e-mail when i order because that e-mail won't be my email. I think that the logout button is really subtle so i would like that it's implemented like the design.